react-graphql-apollo-tutorial.md

```diff
@@ -114,7 +112,7 @@ Simply define your environment variables in this *.env* file. In your *.env* fil
 GITHUB_PERSONAL_ACCESS_TOKEN=xxxXXX
 {{< /highlight >}}
 
-In any Node.js application, you can use the key in your actual source code with the following package: {{% a_blank "dotenv" "https://github.com/motdotla/dotenv" %}}. Follow their instructions to install it for your project. Usually it is only a `npm install dotenv` followed by including `require('dotenv').config()` in your *index.js* file. Afterward use the personal access token from the *.env* file in your *index.js* file. If you run into an error afterward when starting your application, just continue to read this section to fix it.
+In any Node.js application, use the key in your source code with the following package: {{% a_blank "dotenv" "https://github.com/motdotla/dotenv" %}}. Follow their instructions to install it for your project. Usually, the process is only a `npm install dotenv`, followed by including require('dotenv').config() or import 'dotenv/config'; in your index.js file.. Afterward use the personal access token from the *.env* file in your *index.js* file. If you run into an error, just continue reading this section to learn how to fix it.
```

Comment:
` is missing for the two inline code blocks. Maybe just copy and paste it before/after require('dotenv').config() and import 'dotenv/config';

Reply:
Unclear. I think maybe you should add this one after I've done all the others.

Reply:
Okay 👍